doc/project/module.md

```diff
@@ -19,6 +19,8 @@ scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.ESModule) }
 scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
 {% endhighlight %}
 
+Note: when using ECMAScript modules, fullopt file [optimization](../internals/performance.html) is limited, because Google Closure Compiler cannot be used with them.
```

Comment:
For grammar and consistency in naming:
```suggestion
Note: when using ECMAScript modules, the [optimizations performed in `fullLinkJS`](../internals/performance.html) are limited, because the Google Closure Compiler cannot be used with them.
```